### DISCUSSION TITLE
How to get interactive GUI on Windows?

### DISCUSSION THREAD

**Discussion by sdeepak88** [lllyasviel/Fooocus#1488] [Q&A]
Hi, I have recently installed the models on Windows 11 system. It is running fine on webui interface after I click run.bat etc. However, I can't seem to get the nice interactive GUI interface as see on some screenshots. It is helpful in making small edits. Can somebody tell me how to get that?

**Comment by ItsNoted** [comment]
Are you talking about the Input Image section? If you tick that and drag an image from above to that section, you can inpaint, outpaint and more.

**Comment by Xposure345** [comment]
The default web browser just open automatically after a short time. Does your browser not open automatically? Perhaps you need to type in the host manually after launch the run.bat.

**Comment by sdeepak88** [comment]
Thanks for answers. I just had to click advanced check box at the bottom and get that inpaint/outpaint patch. Everything is working fine now.
